.github/workflows/build.yml

```diff
@@ -7,47 +7,78 @@ on:
       - '*'
       - '!master'
 
+env:
+  IMAGE_NAME: ${{ github.repository_owner }}/node-base
+      
 jobs:
   build_matrix:
     strategy:
       matrix:
         # NOTE: These versions must be kept in sync with the release.yml
-        version: ["18.19.1", "20.11.1", "22.2.0"]
+        # In the case where a new node version introduces a breaking change,
+        # replace the major version of the matrix to a pinned version here.
+        # Ex: ["18", "20", "22"] ---> ["18.19.1", "20", "22"]
```

Comment:
Unfortunately I've just realized how this isn't going to achieve what we want.  By going from `18` to `18.19.1`, the `build-push-action` will go from pushing `18` to pushing `18.19.1` and won't update the `18` tag.  I mistakenly thought the code on 51-68 managed the major number as well.